checks.d/docker_daemon.py

```diff
@@ -491,14 +511,34 @@ def _report_container_size(self, containers_by_id):
             tags = self._get_tags(container, PERFORMANCE)
             m_func = FUNC_MAP[GAUGE][self.use_histogram]
             if "SizeRw" in container:
-
                 m_func(self, 'docker.container.size_rw', container['SizeRw'],
                        tags=tags)
             if "SizeRootFs" in container:
                 m_func(
                     self, 'docker.container.size_rootfs', container['SizeRootFs'],
                     tags=tags)
 
+    def _report_container_count_by_state(self, containers_by_id, state="Any"):
+        count = 0
+        tags = {}
+        filterlambda = lambda x: not self._is_container_excluded(x) and state is "Any" or container["State"] is state
```

Comment:
`container["State"]` --> `x["State"]`, or rename x to container and `self._is_container_excluded(x)` --> `self._is_container_excluded(container)`